.github/workflows/tests-no-coverage.yml

```diff
@@ -0,0 +1,58 @@
+name: Execute tests with no coverage (-Werror, gcc13)
+# In secp256k1 some assertions are not exactly equal when coverage is enabled
+# and when it is not. Hence, it is better to run the two cases separately.
+
+on:
+  push:
+    branches:
+      - frost
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # source for using Fedora via a container in the runner:
+  #     https://github.com/avocado-framework/avocado/actions/runs/2730087531/workflow#L15-L32
+  # and
+  #     https://github.com/actions/runner-images/issues/2307#issuecomment-1576493700
+  #
+  # Moreover, we cannot use ${{github.workspace}} variable inside a container,
+  # we must use ${GITHUB_WORKSPACE}, see:
+  #    https://github.com/actions/runner/issues/2058
+  # and
+  #    https://github.com/actions/runner/pull/2517
```

Comment:
is this still a relevant comment? No occurrence of `${GITHUB_WORKSPACE}` in the workflow file.

Reply:
yep! Removed.